.eslintignore

```diff
@@ -1,11 +1,5 @@
-activity-streams-env/
-dist/
-firefox/
-logs/
-stats.json
-prerendered/
+bin/prerender.js                                                                                       
+data/                                                                                                  
+logs/                                                                                                  
+prerendered/                                                                                           
```

Comment:
Some reason these lines have a lot of trailing whitespace ?? I was wondering why the `diff` didn't leave `logs/` unchanged…